server/routerlicious/packages/services-ordering-rdkafka/src/rdkafkaBase.ts

```diff
@@ -42,6 +44,18 @@ export abstract class RdkafkaBase extends EventEmitter {
             replicationFactor: options?.replicationFactor ?? 3,
         };
 
+        // In RdKafka, we can check what features are enabled using kafka.features. If "ssl" is listed,
+        // it means RdKafka has been built with support for SSL.
+        const rdKafkaHasSSLEnabled =
+            kafka.features.filter((feature) => feature.toLowerCase().indexOf("ssl") >= 0).length > 0;
+
+        if (rdKafkaHasSSLEnabled && options?.sslCACertFilePath) {
```

Comment:
Should if throw is `options?.sslCACertFilePath` is provided but the feature is enabled? Otherwise you may think SSL is enabled but it's not actually

Reply:
Did you mean if the user provides `options?.sslCACertFilePath` but SSL is *disabled* (maybe because RdKafka was not built with SSL support)?

Reply:
Yea

Reply:
Got it, that makes sense. I agree with you. Let me send updates soon